bedrock/firefox/templates/firefox/welcome/page10.html

```diff
@@ -45,7 +45,7 @@ <h1 class="c-main-title">{{ ftl('welcome-page10-main-heading') }}</h1>
 
       <div class="mzp-c-picto">
         <div class="mzp-c-picto-image">
-          {% include 'firefox/welcome/includes/page10/laptop.svg' %}
+          <img src="{{ static('img/firefox/welcome/page10/laptop.svg') }}" loading="lazy" alt="">
```

Comment:
**nit:** `loading="lazy"` should come before `src` (see https://bugzilla.mozilla.org/show_bug.cgi?id=1647077)

Same with other occurrences below.

Reply:
Learning something new every day ([MDN: loading#usage_notes](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/loading#usage_notes)) — and in this case of welcomes it's the only engine relevant so, silly me;) I may remove them after all, as they're right around the edge of the fold for various screen sizes, so they may not add any improvement whatsoever…

Q: aren't welcome pages actually downloaded in the background, as per https://support.mozilla.org/kb/firefox-browser-welcome-pages#w_how-do-firefox-browser-welcome-pages-get-delivered? Or is it meant that only their list is downloaded ahead, and the actual rendered pages chosen to display and their assets still load "live", online?

Reply:
I believe it's just the list that gets downloaded 👍

Reply:
In that case I'm leaving it out completely, to rather load the images eagerly, not risking any potential perception of the pages as being "slow" if something gets loaded delayed…